CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.36.0]
+### Changed
```

Comment:
Perhaps this should be "Fixed", and this be made into a patch release? It does seem that PPOM does not work properly when errors are thrown like they were